app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,60 @@
+class PagesController < ApplicationController
+  def home
+    @products = Product.all
+    
+    if params[:category] == 'entree'
+      @products = @products.where(category: 0)
```

Comment:
since you've defined an enum, you can simply use that name either as a scope or as a query
eg `Product.second_course` or `Product.where(category: :second_course)`

And since you have this you can simply transform all this code into 
```ruby
    @products = Product.all
     @products = @products.where(category: params[:category]) if params[:category]
```